_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mesh Expansion
+overview: Instructions to add external machines and expanding Istio.
+
+order: 60
+
+layout: docs
+type: markdown
+---
+
+Instructions to configure Istio on a Kubernetes Cluster so it can be expanded with
+services running on cloud or on-prem VMs or external machines.
+
+## Prerequisites
+
+* You have [installed Istio](quick-start.html) on Kubernetes.
+
+* The machine must have IP connectivity to the endpoints in the mesh. This
+typically requires same VPC or a VPN connection, as well as a container network that
+provides direct (without NAT or firewall deny) routing to the endpoints. The machine
+is not required to have access to the cluster IP addresses assigned by K8S.
+
+* The control plane (Pilot, Mixer, CA) and Kubernetes DNS server must be accessible
+from the VMs. This is typically done using an [Internal Load
+Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
+Other options are possible, for example NodePort, or running Istio components on VMs, or
+custom network configurations - separate documents will cover advanced configs.
+
+## Installation steps
+
+Setup consists of preparing the mesh for expansion and installing and configuring each VM.
+
+An example script to help with K8S setup is available in
+[install/tools/setupMeshEx.sh](https://raw.githubusercontent.com/istio/istio/master/install/tools/setupMeshEx.sh).
+
+An example script to help configure a machine is available in [install/tools/setupIstioVM.sh](https://raw.githubusercontent.com/istio/istio/master/install/tools/setupIstioVM.sh).
+You should customize it based on your provisioning tools and DNS requirements.
+
+### Preparing the K8s for expansion
+
+* Setup internal load balancers for Kube DNS, Pilot, Mixer and CA. This step is specific to
+each cluster, you may need to add annotations.
```

Comment:
Because this setup process is specific to each cluster, you might need to add annotations.